cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/constraint/LastRunConstraint.java

```diff
@@ -30,19 +30,19 @@
  */
 public class LastRunConstraint extends ProtoConstraint.LastRunConstraint implements CheckableConstraint {
 
-  public LastRunConstraint(long millisSinceLastRun) {
-    super(millisSinceLastRun);
+  public LastRunConstraint(long secsSinceLastRun) {
```

Comment:
this should still be `millisSinceLastRun`. See `SchedulerBuilder#withDurationSinceLastRun(long delayMillis)`; this is the API that users will use.